examples/tracetest-lightstep/tracetest/tracetest-provision.yaml

```diff
@@ -1,2 +1,27 @@
-dataStore:
+---
+type: PollingProfile
+spec:
+  name: Default
+  strategy: periodic
+  default: true
+  periodic:
+    retryDelay: 500ms
+    timeout: 30s
```

Comment:
Same thing here. This profile is stricter than the [default profile](https://github.com/kubeshop/tracetest/blob/main/server/executor/pollingprofile/polling_profile_resource.go#L29). It makes sense to use the default instead?